k8s/agent-resources-supertenant.yaml

```diff
@@ -15,6 +19,9 @@ spec:
     metadata:
       labels:
         app: logdna-agent
```

Comment:
any reason to leave this in the DaemonSet definitions?

Reply:
Only that it's used in the matchLabels (which could easily be updates) and that someone's tooling might be expecting it. But yeh, I believe "app" has been deprecated in favour of app.kubernetes.io/name. I'm not opposed to removing it if you think the tidiness is worth it

Reply:
we should clean it up at some point IMO, but removing labels might need to be considered a major/breaking change for the reason you mentioned. I'm fine with it as-is here, just stood out in these changes.

Reply:
I've added a ticket to the brand new rust/native agent v3 epic